.terraform-docs.yaml

```diff
@@ -18,3 +19,7 @@ sort:
 output:
   file: README.md
   mode: replace
+
+recursive:
+  enabled: true
+  path: examples
```

Comment:
Need to revert my own change since the terraform-docs recursive path feature this won't be supported by the pre-commit hook ([reference](https://github.com/antonbabenko/pre-commit-terraform/issues/439))